Week3/homework/1-step3.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+    // What to do here? 
+    func();
 }
 
 function bar() {
-  console.log('Hello, I am bar!');
+    console.log('Hello, I am bar!');
 }
 
```

Comment:
I would return the value of calling func(). foo should know nothing about the function it is calling, and so should not make assumptions about it.  Usually one runs a function to use its return value.